### PR TITLE
Rdt target group to target user

### PR DESCRIPTION
**Pull request by martaswiatkowska**
Renamed target_group to Target_user and added new objects to target_user as is specified in new Resource  Profile

### CODE REVIEW

**Comment by agpul** [comment]
I can't reach this PR, @wziajka can you check?

File: lib/tasks/rdt.rake [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
```
[the remark this answers is not in the document]
Comment (martaswiatkowska) [reply lib/tasks/rdt.rake:1]: I think that this task is important to add this data to production db

**Comment by agpul** [comment]
How can I test it?

**Comment by agpul** [comment]
Shouldn't we change the name in Backoffice from Dedicated for to Target user?

**Pull request merged**